### ISSUE TITLE
HPU ID register

### ISSUE THREAD

**Issue by francescodiotalevi** [event-driven-robotics/HPU_Core#7]
Be carefull that the ID of HPU has been changed in B01...
As soon as possible it must be updated again in HPU.

**Comment by mcasti-iit** [comment]
Yes, it is just to note that the Core used in some implementations is NOT a release, but just a Beta Version.
It should be useful to make the driver capable to ignore the HPU Core version if the name starts with B.
